src/types.rs

```diff
@@ -16,7 +16,24 @@ pub type EdgeId = sys::EdgeId;
 /// Type of base mesh for which the textures are defined.  A mesh
 /// can be triangle-based (with triangular textures) or quad-based
 /// (with rectangular textures). */
-pub type MeshType = sys::MeshType;
+#[repr(u32)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum MeshType {
+    Quad = ptex_sys::MeshType::Quad.repr,
```

Comment:
Hmm, one of the effects this has is these values get evaluated and end up showing up in the docs as their raw value.
Which is a bit unfortunate.

pub enum MeshType {
    Quad = 1,
    Triangle = 0,
}